### PR TITLE
Menu branch

### PR DESCRIPTION
**Pull request by Keethan03**

### CODE REVIEW

File: src/components/home/Home.module.css [hunk 1]
```diff
@@ -2,4 +2,7 @@
   width: 100%;
   font-size: large;
   color: red;
+}
+.footer{
+  padding-top: 200px
```
Comment (janacm) [inline src/components/home/Home.module.css:7]: Hey avoid using `px`. Stick with `%`.

File: src/components/menu/Menu.component.tsx [hunk 2]
```diff
@@ -1,11 +1,65 @@
-import React from 'react';
+import React from "react";
+import img1 from './images/menu1.png';
```
Comment (janacm) [inline src/components/menu/Menu.component.tsx:2]: Make sure variable names describe what is contained inside them

File: src/components/menu/Menu.component.tsx [hunk 3]
```diff
@@ -1,11 +1,65 @@
...
+   const image1 = "https://www.notion.so/janac/Menu-Page-dishes-c97e397570384b69a4ce291d02dde758#88140f350d3e40d5986c2a7791ae3709";
   return (
-    <div>
-        You like snacks?Yes!
-        <Footer/>
+
+  <body>
+
+  <img src = {header}
+   style = {{
```
Comment (janacm) [inline src/components/menu/Menu.component.tsx:16]: Avoid using inline styles - move all of these to `Menu.module.css` and add a class for each of these divs instead. Reference that class in your css file. This helps with maintainability and reusability

File: src/components/menu/Menu.component.tsx [hunk 4]
```diff
@@ -1,11 +1,65 @@
...
+    padding: "130px 50px 0px 115px",
+  position: "relative"}}
+    
+    >
+
+    <a href = {img2}><img src = {img2} className = {style.image2}/></a>
+
+  </div>
+  
+
```
Comment (janacm) [inline src/components/menu/Menu.component.tsx:42]: Just have one space between elements, not 2-3 please :)

File: src/components/menu/Menu.module.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+.image2{
+    width: 600px;
+    height: 475px;
+    padding: "100px 50px 0px 105px";
+
+}
+
+.image1:hover{
+
+transform: scale(1.33);
```
Comment (janacm) [inline src/components/menu/Menu.module.css:17]: Fix indentation here

**Review by janacm — CHANGES_REQUESTED** [review]
Hey, the code works but there are a few best practices and code-style issues that you should fix up before we merge this branch in. Make sure you have the VS Code extension "prettier" installed, lmk if you don't.

**Comment by Keethan03** [comment]
Alright sounds good! And no I do not have the VS code extension "prettier"
installed.

**Review by TheVedhaGuy — APPROVED** [review]

**Pull request merged**